erpnext_documentation/www/docs/user/manual/en/accounts/subscription.md

```diff
@@ -16,12 +16,13 @@ Before creating and using a Subscription, it is advisable to create the followin
 1. [Subscription Plan](/docs/user/manual/en/accounts/subscription-plan)
 
 ## 2. How to set a Subscription
-1. Go to the Subscription list and click on New. 
-1. Select a Customer.
+1. Go to the Subscription list and click on New.
+1. Select Party Type as 'Customer' or 'Supplier' and select the party.
 1. Set the Start Date from when the subscription will be active.
+1. Optionally you can also enter the subscription end date if you know it before hand.
 1. Days Until Due is the number of days within which Customer has to pay a generated Sales Invoice.
 1. Select the [Subscription Plans](/docs/user/manual/en/accounts/subscription-plan).
-1. Save. 
+1. Save.
  ![Subscription](/docs/assets/img/accounts/subscription.png)
 
 Based on the subscription start and end date, actual dates for invoices will be calculated.
```

Comment:
@deepeshgarg007 show an actual name in subscription-trial.png instead of test customer